Firmware/OpenSteamController/src/eeprom_access.c

```diff
@@ -158,8 +158,9 @@ int eepromCmdFnc(int argc, const char* argv[]) {
 		uint32_t num_words = strtol(argv[4], NULL, 0);
 		retval = eepromReadToConsole(word_size, addr, num_words);
 	} else if (!strcmp("write", argv[1])) {
-		//TODO: imeplement this
-		printf("eeprom writing not implemented yet\n");
+		printf("experimental eeprom write\n");
+		uint32_t arr[1] = { strtol(argv[4], NULL, 16) };
```

Comment:
Use better variable name than "arr." 

Also variable does not need to be array since it is only 1 32-bit word. You can pass reference to it when calling eepromWrite.